aurora/contracts/src/AuroraErc20FastBridge.sol

```diff
@@ -72,6 +72,7 @@ contract AuroraErc20FastBridge is Initializable, UUPSUpgradeable, AccessControlU
     event FastBridgeWithdrawOnNear(string token, uint128 amount);
     event InitTokenTransfer(
         address indexed sender,
+        address indexed tokenAddress,
```

Comment:
Is it an Ethereum Token Address or an Aurora Token Address?

Reply:
ea6fb880843ff82aecfa5bc385d9cabb50bd0d0c